### PR TITLE
Initial pass at some AWS Certification advice

### PR DESCRIPTION
**Pull request by duncanmadetech**

### CODE REVIEW

File: guides/cloud/aws_certification_advice.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+### AWS Certified Solutions Architect - Professional
+
+The [AWS Certified Solutions Architect - Professional](https://aws.amazon.com/certification/certified-solutions-architect-professional) is the professional level Solutions Architect Certification and probably the most difficult AWS Exam. It covers the following domains. Design for Organisational Complexity, Design for New Solutions, Migration Planning, Cost Control and Continuous Improvement for Existing Solutions.
+
+#### Recommended Online Training
+1. [Cantrill.io](https://learn.cantrill.io/p/aws-certified-solutions-architect-professional)
+2. [ACloudGuru](https://learn.acloud.guru/course/aws-certified-solutions-architect-professional/dashboard)
+
+
+## AWS Speciality Certifications
```
Comment (lukemorton) [inline guides/cloud/aws_certification_advice.md:99]: We may want to hold back on specifying these until we know what we want to prioritise based on propositions and aligning to competencies that support Jonathans efforts to grow our partner offerings.

File: guides/cloud/aws_certification_advice.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+| Exam | Cost | Cost inc tax |
+| --- | --- | --- |
+| AWS Cloud Practitioner | $100 | $120 |
+| AWS Associate Level (Developer, SysOps, Architect) | $150 | $180 |
+| AWS Professional Level (DevOps Engineer, Architect) | $300 | $360 |
+| AWS Speciality Exams (Security, Networking, Database, ML, Data) | $300 | $360 |
+
+Each time you pass an AWS Exam you receive a [free practice exam and 50% off voucher](https://www.certmetrics.com/amazon/candidate/benefit_summary.aspx) for the next exam.
+
+This means an individual could achieve all 11 AWS Certifications for $1650!
```
Comment (lukemorton) [inline guides/cloud/aws_certification_advice.md:28]: Is this good or bad? I can't tell.

**Review by lukemorton — APPROVED** [review]
Looking good, just a few small comments you can take or leave.

File: guides/cloud/aws_certification_advice.md (hunk 2 above)
Comment (duncanmadetech) [reply guides/cloud/aws_certification_advice.md:28]: I think it's pretty good, some one day training courses cost that

File: guides/cloud/aws_certification_advice.md (hunk 1 above)
Comment (duncanmadetech) [reply guides/cloud/aws_certification_advice.md:99]: Sure, this was just a first pass/WIP. I was going to pad out the speciality areas in a second pass :-) you know how agile I am :-)

**Pull request merged**